src/detect-engine-tls.c

```diff
@@ -341,3 +341,45 @@ int DetectEngineInspectTlsSubject(ThreadVars *tv, DetectEngineCtx *de_ctx,
 
     return cnt;
 }
+
+int DetectEngineInspectTlsGeneric(ThreadVars *tv, DetectEngineCtx *de_ctx,
```

Comment:
This is a copy of DetectEngineInspectGenericList from detect-engine-dns.c right? Might be better to move that one in to detect-engine.c and call that.